stdlib/array.pyi

```diff
@@ -33,7 +33,10 @@ class array(MutableSequence[_T], Generic[_T]):
     def fromfile(self, __f: BinaryIO, __n: int) -> None: ...
     def fromlist(self, __list: List[_T]) -> None: ...
     def fromunicode(self, __ustr: str) -> None: ...
-    def index(self, __v: _T) -> int: ...  # type: ignore  # Overrides Sequence
+    if sys.version_info >= (3, 10):
+        def index(self, __v: _T, start: int = ..., stop: int = ...) -> int: ...
```

Comment:
```suggestion
        def index(self, __v: _T, __start: int = ..., __stop: int = ...) -> int: ...
```

```
array('f', [1.0, 2.0, 3.0])
>>> a.index(1, start=2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: array.index() takes no keyword arguments
```

Reply:
Maybe we should enable stubtest on Python 3.10?

Reply:
Yeah, we should. My flurry of changes will make that easier, since there will be less allowlist-finagling.